zkevm/tests/integration.rs

```diff
@@ -122,9 +121,25 @@ fn test_mock_prove_all_with_circuit<C: TargetCircuit>(
     failed_cases
 }
 
+#[cfg(feature = "prove_verify")]
+#[test]
+fn test_mock_prove_all_target_circuits_packing() {
+    use zkevm::circuit::EvmCircuit;
+
+    init();
+    let mut block_results = Vec::new();
+    for block_number in 1..=15 {
+        let trace_path = format!("tests/traces/bridge/{:02}.json", block_number);
+        let block_result = get_block_result_from_file(trace_path);
+        block_results.push(block_result);
+    }
+    Prover::mock_prove_target_circuit_packing::<EvmCircuit>(&block_results, true).unwrap();
```

Comment:
@noel2004  can you add zktrie and poseidon here and make sure they pass?

Reply:
ok

Reply:
Test has passed in `test_mock_prove_all_target_circuits_packing`:

(I add logs for rows being used)

>[2022-08-29T05:14:40Z INFO  zkevm::prover] start mock prove zktrie
>[2022-08-29T05:14:40Z INFO  zkevm::circuit] zktrie use rows 1766
>[2022-08-29T05:14:42Z INFO  zkevm::prover] mock prove zktrie done
>[2022-08-29T05:14:42Z INFO  zkevm::prover] start mock prove poseidon
>[2022-08-29T05:14:42Z INFO  zkevm::circuit] poseidon use rows 33250
>[2022-08-29T05:14:51Z INFO  zkevm::prover] mock prove poseidon done